Assets/Scripts/Prototype/ProjectilePrototype/SpawnProjectile.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.InputSystem;
+using UnityEngine.InputSystem.Controls;
+
+public class SpawnProjectile : MonoBehaviour
+{
+    public static bool didProjectileHit = false;
+
+    public GameObject firePoint;
+    public List<GameObject> vfx = new List<GameObject>();
+    public Camera currentCamera;
+    public AudioSource shootSound;
+    public AudioSource impactSound;
```

Comment:
Check `AudioManager.cs` and setup sfx in there instead. That would make audio playing code a lot cleaner.

Reply:
okay 👍